assets/src/admin.js

```diff
@@ -60,26 +60,86 @@ jQuery(function ($) {
 		//make sure geocoding is finished (basic form validation)
 		var form_valid = true;
 
```

Comment:
Sorry, this is where I went nuts. Made an object to track all the fields, then a small jQuery function to track / attach "state" to each field, and an upper formstate check, so `formatted_address` and `conference_url` (and other fields) could have issues independently.